firefly/kernel/init/init.cpp

```diff
@@ -76,8 +76,7 @@ __attribute__((section(".stivale2hdr"), used)) static stivale2_header stivale_hd
 
 // We will now write a helper function which will allow us to scan for tags
 // that we want FROM the bootloader (structure tags).#include <cstdlib/#include <cstdint>>
-void* stivale2_get_tag(stivale2_struct* stivale2_struct, uint64_t id)
-{
+void* stivale2_get_tag(stivale2_struct* stivale2_struct, uint64_t id) {
```

Comment:
Document what this returns, potentially represent the `tag` in a struct and return that?

Reply:
It is documented, isn't it?

> potentially represent the tag in a struct and return that
I'm not sure what you mean by that.
This isn't our code FYI, it's how we interface with the bootloader to query information such as a framebuffer or memory map

Reply:
It is documented, isn't it?

> potentially represent the tag in a struct and return that

I'm not sure what you mean by that.
This isn't our code FYI, it's how we interface with the bootloader to query information such as a framebuffer or memory map